tube.js

```diff
@@ -97,9 +100,15 @@ export class Tube {
         if (this.debug) console.log("host write " + utils.hexword(addr) + " = " + utils.hexbyte(b));
         switch (addr & 7) {
             case 0:
-                if (b & 0x80) this.r1stat |= b & 0x3f;
-                else this.r1stat &= ~(b & 0x3f);
-                this.hstat[0] = (this.hstat[0] & 0xc0) | (b & 0x3f);
+                if (b & 0x80)
+                this.r1stat |= b & 0x3f;
```

Comment:
Can you indent this line, or combine it onto the previous line please? This is a confusing indent.

Reply:
Indented